images/docker/setup_systemd.sh

```diff
@@ -21,6 +21,9 @@
 
 set -eux
 
+yum install -y initscripts
+rm -rf /var/cache/yum
```

Comment:
@dmitrydb  could you move that line to the final end?

Reply:
Yes, done